lib/ansible/playbook/helpers.py

```diff
@@ -95,6 +95,10 @@ def load_list_of_tasks(ds, play, block=None, role=None, task_include=None, use_h
     from ansible.playbook.role_include import IncludeRole
     from ansible.playbook.handler_task_include import HandlerTaskInclude
     from ansible.template import Templar
+    from ansible.utils.plugin_docs import get_versioned_doclink
+
+# we'll need this down here
+    include_link = get_versioned_doclink('user_guide/playbooks_reuse_includes.html')
```

Comment:
move to inside the else, since it is the only place where it is used (avoid doing work when not needed)